docs/web/index.md

```diff
@@ -11,8 +11,7 @@ Features:
   - [Jackson Object Mapping](#jackson)
   - [Monitoring](#monitoring)
   - [Tracing](#tracing)
-  - [Health Checks](#health-checks)
-  - [Testing](#testing)
+  - [Health Checks](#health-checks--actuator)
```

Comment:
The bullet point list does not include "security". Should we add it?

Reply:
Imho not. These links are just anchors within the page. I would not mix it with links to different pages. I added the security page to [the mkdocs](https://github.com/SDA-SE/sda-spring-boot-commons/pull/163/commits/fb3075e18b65c3ebcfdd5375ee48a18e1480ddb5). Whenever we provide github pages or Backstage docs, it will be visible in the main nav. But that's a separate PR.